src/services/configService.js

```diff
@@ -69,4 +63,22 @@ export class ConfigService {
     return HttpClient.put(`${ENDPOINT}solution-settings/logo`, logo, options)
       .map(prepareLogoResponse);
   }
+
+  /* get solution settings */
+  static getSolutionSettings() {
+    return HttpClient.get(`${ENDPOINT}solution-settings/theme`)
+      .map(toSolutionSettingThemeModel)
+      /* When the application loads for the first time, there won't be "solution-settings"
+       in the storage. In that case, service will throw a 404 not found error.
+       But we need to ignore that and stick with the static values defined in UI state */
+      .catch(error =>
+        error.status === 404 ? Observable.of(true) : Observable.throw(error)
+      );
+  }
+
+  /* update solution settings */
+  static updateSolutionSettings(model) {
+    return HttpClient.put(`${ENDPOINT}solution-settings/theme`, model)
```

Comment:
Theme?

Reply:
no

Reply:
I thought you were asking if it isn't "Theme" in camel case. The URL has a weird name in API side. But this is the correct one.